data/hai/hai reveal 3 early.txt

```diff
@@ -429,4 +432,78 @@ event "role-play station active"
 		security 0.7
 		"required reputation" -10
-	
+
+
+
+# This begins with a P instead of an H so it will happen after the other Hai Reveal mission's dialog.
+mission "Pirates with Hai Weapons 1: Katya's Message"
+	landing
+	name "Pirates with Hai Weapons"
+	description "Katya wants you to find out if the Hai government is arming pirates, and if they are, to stop them. Don't tell the Hai that the Free Worlds sent you, and make sure the Republic doesn't find out."
+	source Hai-home
+	destination Hai-home
+	to offer
+		has "Hai Building Mountaintop [1b] Return: done"
+	to complete
+		has "Hai Reveal [A05] Take Teeneep to Hai-home: done"
+	on offer
+		conversation
+			`Before you leave, Alondo finds you in a hallway. "Captain <last>, come with me. We need to have a quick conference." You follow him to an unusual room with strange padding on all sides. He plugs a device with an attached keyboard into a wall-mounted monitor and types for a few seconds. Soon after, you see the familiar face of Katya.`
+			`	"The transmission is secure," Alondo says. "This room has good soundproofing and some signal dampening. You can talk freely."`
+			choice
+				`	"What is it, Katya?"`
+					goto whaizzit
+				`	"You've taken a lot of precautions for this call."`
+			`	"Yes," she says. "This is a sensitive matter.`
+			label whaizzit
+			`	"There's a problem I need you to solve... discretely. The Free Worlds have seen pirates with Hai weapons for quite a while. Most weapons were on ships, but recently we've found a large stock in a warehouse on an anarchist world.`
+				to display
+					not "FW Bounty 3: done"
+			`	"There's a problem I need you to solve... discretely. The Free Worlds have seen pirates with Hai weapons for quite a while. I know you fought the pirate ship Silverhawk back before the war. We know now that it had Hai outfits. Recently we've found a large stock in a warehouse on an anarchist world.`
+				to display
+					has "FW Bounty 3: done"
+			`	"Such a vast supply is unlikely to have come from individual smugglers, so we must assume it's the result of a large-scale operation. I'm worried the Hai government may be arming pirates. If they are, I hope the weapons are just misguided tribute payments. In the south, we well know how useful bounties and tributes are when dealing with pirates, but from my time as a councilor, I can tell you the Republic will not be so understanding.`
+			`	"You must find out if the Hai government is arming the pirates and, if they are, put a stop to it. Don't tell them the Free Worlds asked you to do it and, whatever you do, don't let the Republic know. It could lead to an explosive diplomatic incident."`
```

Comment:
```suggestion
			`	"You must find out if the Hai government is arming the pirates and, if they are, put a stop to it. Don't tell them the Free Worlds asked you to do it, and whatever you do, don't let the Republic know. It could lead to an explosive diplomatic incident."`
```
It could be said both ways, but my suggestion should read better and still sound right.